debian/xenial/kubeadm/etc/systemd/system/kubelet.service.d/10-kubeadm.conf

```diff
@@ -0,0 +1,8 @@
+[Service]
+Environment="KUBELET_KUBECONFIG_ARGS=--kubeconfig=/etc/kubernetes/kubelet.conf --require-kubeconfig=true"
+Environment="KUBELET_SYSTEM_PODS_ARGS=--pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true"
+Environment="KUBELET_NETWORK_ARGS=--network-plugin=cni --cni-conf-dir=/etc/cni/net.d --cni-bin-dir=/opt/cni/bin"
+Environment="KUBELET_DNS_ARGS=--cluster-dns=100.64.0.10 --cluster-domain=cluster.local"
```

Comment:
Basically the default CIDR in `kubeadm` is `100.64/12`, DNS gets 10th IP, and we have to tell kubelet about it. User may override the CIDR and the domain in `kubeadm init`, but currently it remains their responsibility to fix the kubelet flag. We could consider adding another drop in to aid this, but may be it's better to document that part, as it's not expect to be so common to change these settings.